### DISCUSSION TITLE
Bookmark reminders

### DISCUSSION THREAD

**Discussion by Guuri11** [dailydotdev/daily#889] [Feature Requests]
- Once you save a post as a bookmark, you'll be prompted if you want to set a reminder to read later
- if you decide to set a reminder, you can select a specific date and time to receive a notification on your mobile, browser or email
- if you don't, you can do so later by accessing the bookmark section and selecting set reminder
- you can receive periodic reminders to review your pending bookmarks

With this feature, people will be able to keep bookmarks organized and remember to read them at the right time, which will help people like me to stay up-to-date with reading and prevent them from pilling up

**Comment by idoshamun** [comment]
That's a great idea. Thanks for the suggestion! 🙏

**Comment by Devanshukoli** [comment]
Hey! I have been using this feature a lot. And find need to reminder long dates. 
- What I mean is, like **Custom Dates**. It would be blessing. I have habbit of organizing things before I jump into it. So this will be really helpful. 
- since in weekends I like to wander out and check new things here and there.


### Currently We do not have this, and max you can put reminder till next week. ( I love this too. )
https://github.com/user-attachments/assets/fe2e515e-7091-4281-bde0-e572feb3f9b1

**Reply by idoshamun** [reply]
Thanks for the suggestion! We will consider it
